src/wdio/services/TerraCommands/hide-input-caret.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Hides the blinking input caret that appears within editable text areas to prevent inconsistent test failures.
+ *
+ * @param {string} selector The selector for the element to hide the caret of
+ */
+const hideInputCaret = (selector) => {
+  if (!global.browser.isExisting(selector)) {
+    throw new Error(`No element could be found with the selector '${selector}'.`);
```

Comment:
Can we update to use toolkit's Logger instead of throwing our own error?
example: https://github.com/cerner/terra-toolkit/blob/master/src/wdio/services/SeleniumDockerService.js#L74

Reply:
Changed in cce5091cbdebce172e5f5c54d0950f86134a8b33